packages/main/src/plugin/color-registry.ts

```diff
@@ -1149,5 +1149,11 @@ export class ColorRegistry {
       dark: colorPalette.gray[500],
       light: colorPalette.gray[800],
     });
+    // Scaled / updated, use blue as it's a 'neutral' color
+    // to indicate that it's informative but not a problem
+    this.registerColor(`${status}updated`, {
```

Comment:
Can we use a more generic term for the color name, ie `status-neutral`, so we can use it for other statuses than `updated`?